prac_07/README.md

```diff
@@ -1 +1,8 @@
 # Practical 07
+This is practical 07. [Practical 07](https://github.com/CP1404/Practicals/tree/master/prac_07)
+
+## In readme.md include
+ 7 different files.
+
+* I don't really get this time about datetime need to learn more
+* There may have error.
```

Comment:
I think in this file is supposed to be the main for the whole repository, so outside the prac 7 file, and inside should be a short introduction of yourself and what the file is about